be/src/runtime/tablets_channel.cpp

```diff
@@ -219,7 +218,7 @@ Status TabletsChannel::reduce_mem_usage(TabletWriterAddResult* response) {
         if (_state == kFinished) {
             // TabletsChannel is closed without LoadChannel's lock,
             // therefore it's possible for reduce_mem_usage() to be called right after close()
-            return _close_status;
+            return;
```

Comment:
Add a log here? this case should not happen frequently

Reply:
done.